recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,243 @@
+            "ec2",
+            "ecr",
+            "ecs",
+            "eks",
+            "elasticache",
+            "elasticbeanstalk",
+            "elasticfilesystem",
+            "elasticloadbalancing",
+            "elasticloadbalancingv2",
+            "elasticmapreduce",
+            "elastictranscoder",
+            "email",
+            "es",
+            "events",
+            "firehose",
+            "fms",
+            "fsx",
+            "gamelift",
+            "glacier",
+            "globalaccelerator",
+            "glue",
+            "greengrass",
+            "guardduty",
+            "health",
+            "iam",
+            "identity_management",
+            "importexport",
+            "inspector",
+            "iot_data",
+            "iot_jobs_data",
+            "iot",
+            "kafka",
+            "kinesis",
+            "kinesisanalytics",
+            "kinesisvideo",
+            "kms",
+            "lambda",
+            "lex",
+            "lightsail",
+            "logs",
+            "machinelearnings",
+            "macie",
+            "marketplace_entitlement",
+            "marketplacecommerceanalytics",
+            "mediaconvert",
+            "medialive",
+            "mediapackage",
+            "mediastore",
+            "mediatailor",
+            "meteringmarketplace",
+            "mobileanalytics",
+            "monitoring",
+            "mq",
+            "mturk_requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource_groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text_to_speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = merge_dicts_for_sdk({"build_" + x: [True, False] for x in sdks}, {
+            "shared": [True, False],
+            "fPIC": [True, False],
+            "min_size": [True, False]
+        })
+    default_options = merge_dicts_for_sdk({"build_" + x: False for x in sdks}, {
+            "shared": False,
+            "fPIC": True,
+            "min_size": False
+        })
```

Comment:
Why add support for py < 3.4? --> https://docs.conan.io/en/latest/installation.html
```
Python 3.4 support has been dropped and Python 2 is being deprecated.
...
Starting in Conan 1.22, Python 2 is no longer supported and Conan will stop working with it in the following releases.
```
So why not use https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression-in-python-taking-union-o?
```
options = {
    **{"build_" + x: [True, False] for x in sdks},
    **{"shared": [True, False], "fPIC": [True, False], "min_size": [True, False]},
}
```

Reply:
Done here: ee448465